cross_list_courses/static/cross_list_courses/js/controllers/ListController.js

```diff
@@ -60,13 +60,15 @@
                         next: '',
                         previous: '',
                     },
+                    processing: '<img src="' + window.globals.STATIC_URL + 'images/ajax-loader-large.gif" class="loading"/> Loading...',
```

Comment:
Went by what the PO pointed at (in the Bulk Site Creator), didn't recollect that we had the fontawesome one.  Happy to switch, but she wanted the large spinner as opposed to the smaller one. I can check in with Chrissy and @Vittorio2015 and switch. Thanks for pointing it out.